src/NotificationTarget.php

```diff
@@ -561,6 +561,26 @@ public static function updateTargets($input)
         }
     }
 
+    public function post_purgeItem()
+    {
+        parent::post_deleteItem();
+        $this->addNotificationTargets($this->entity);
+        if (
+            count($this->target) == 0 &&
+            Session::haveRight("config", UPDATE) &&
+            isset($this->notification_targets["1_1"]) &&
+            $this->notification_targets["1_1"] === "1_" . __("Administrator")
+        ) {
+            $notification = new Notification();
+            $notification->getFromDB($this->fields['notifications_id']);
+            self::updateTargets([
+                'notifications_id' => $this->fields['notifications_id'],
+                '_targets'         => ["1_1"],
+                'itemtype'         => $notification->fields['itemtype']
+            ]);
+        }
+    }
+
```

Comment:
If I understand correctly, when the user chooses to remove all recipients, it forces the addition of the administrator?
This might be misunderstood and seen as a bug, as some users might intentionally want to remove all recipients.
I think we should suggest a default recipient for fresh install only without altering the existing configurations.